ihmc-high-level-behaviors/src/main/java/us/ihmc/PerceptionAndAutonomyProcess.java

```diff
@@ -644,12 +682,16 @@ public static void main(String[] args)
       ROS2Node ros2Node = ROS2Tools.createROS2Node(CommunicationMode.INTERPROCESS.getPubSubImplementation(), "perception_autonomy_process");
       ROS2Helper ros2Helper = new ROS2Helper(ros2Node);
 
+      RapidHeightMapExtractor.reloadHeightMapParameters("GPUBuildingExploration"); // FIXME: This should be redesigned.
```

Comment:
@PotatoPeeler3000 This was the only way to load a different parameter set unfortunately. Why the heck are they statically accessed everywhere :cry: